release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3023,6 +3023,36 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.18.19
+[id="ocp-4-18-19_{context}"]
+=== RHSA-2025:9725 - {product-title} {product-version}.19 bug fix and security update
+
+Issued: 2 July 2025
+
+{product-title} release {product-version}.19 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:9725[RHSA-2025:9725] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2025:9726[RHSA-2025:9726] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.18.19 --pullspecs
+----
+
+[id="ocp-4-18-19-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the resources containing the configuration for the vSphere connection would break because of a mismatch between the user interface and API. With this release, the resources do not break because the user interface uses the updated API definition. (link:https://issues.redhat.com/browse/OCPBUGS-57580[OCPBUGS-57580])
+
+* Previously, the `oc adm node-image create` command incorrectly modified the existing permissions of the target assets folder when the command saved the artifacts on the disk. With this release, a bug fix ensures that the copying operation for the command keeps the destination folder permissions. (link:https://issues.redhat.com/browse/OCPBUGS-57507[OCPBUGS-57507])
+
+* Previously, using Alertmanager apiVersion v1 in the `openshift-monitoring/cluster-monitoring-config` or `openshift-user-workload-monitoring/user-workload-monitoring-config` parameters caused {product-title} 4.19 to fail early with an `InvalidConfigXXX` error. This issue occurred because {product-title} 4.19 is using Prometheus v3 which does not support Alertmanager apiVersion v1. With this release, {cmo-first} sets `upgradable=false` if apiVersion v1 is detected in the config maps to prevent the `InvalidConfigXXX` error from occurring after upgrading to {product-title} 4.19. As a result, clusters upgrade through {product-title} 4.18.x before moving to {product-title} 4.19. (link:https://issues.redhat.com/browse/OCPBUGS-56251[OCPBUGS-56251])
```

Comment:
```suggestion
* Previously, using Alertmanager apiVersion v1 in the `openshift-monitoring/cluster-monitoring-config` or `openshift-user-workload-monitoring/user-workload-monitoring-config` parameters caused {product-title} 4.19 to fail early with an `InvalidConfigXXX` error. This issue occurred because {product-title} 4.19 is using Prometheus v3 which does not support Alertmanager apiVersion v1. With this release, {cmo-first} sets a value of `upgradable=false` if apiVersion v1 is detected in the config maps to prevent the `InvalidConfigXXX` error from occurring after upgrading to {product-title} 4.19. As a result, clusters upgrade through {product-title} 4.18.x before moving to {product-title} 4.19. (link:https://issues.redhat.com/browse/OCPBUGS-56251[OCPBUGS-56251])
```